pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -159,6 +159,8 @@ def _call_children_scripts(self):
             if _HYDRA_AVAILABLE:
                 if HydraConfig.initialized():
                     cwd = get_original_cwd()
+                    os_cwd = f'"{os.getcwd()}"'
```

Comment:
```suggestion
                    os_cwd = os.getcwd()
```

Reply:
Hey @kaushikb11, keeping ` os_cwd = f'"{os.getcwd()}"`' change is actually necesarry for cases where ${hydra.job.override_dirname} is used as part of the directory name, for example:
```
hydra:
  run:
    dir: ${log_dir}/runs/${now:%Y-%m-%d}/${now:%H-%M-%S}_${hydra.job.override_dirname}
```
without this change hydra thinks you are trying to override the run_dir with invalid characters that are used as separators (commas and = symbols). I know it doesn't look pretty, but I don't have a better solution right now.

Reply:
This was purposefully set as if the user has set the directory to include some form of interpolation, we need to make sure this is evaluated as a string.

We should add a test case for this eventually.